x-pack/packages/kbn-elastic-assistant/impl/assistant/api/index.tsx

```diff
@@ -40,9 +38,7 @@ export interface FetchConnectorExecuteResponse {
 
 export const fetchConnectorExecuteAction = async ({
   conversationId,
-  isEnabledRAGAlerts,
```

Comment:
We removed `isEnabledRAGAlerts` from the API, but in UI we still have the toggle possible to turn off. 
<img width="787" alt="Screenshot 2024-07-12 at 11 46 49 AM" src="https://github.com/user-attachments/assets/05117f74-f787-42de-a2a5-e14c5aa842e6">

I think we should be consistent and remove it with leaving only UI ability to manage a number of alerts.. 
Also telemetry has it hardcoded to true. It will be confusing to provide the UX to manage it availability.

Reply:
Whoops! Yep, will remove this.

I'm going to keep the local storage state as-is for now, since it also holds the alert count setting from this dialog. We should figure out where we want to store this long-term. Not sure if this should be a user setting or could be global -- will bring up with @jamesspi.

Reply:
Resolved in [27125f5](https://github.com/elastic/kibana/pull/188168/commits/27125f512afa183e86552d9ad9b868ecd270e799).